### DISCUSSION TITLE
File limit AWS Amplify - Any way to Clear Cache?

### DISCUSSION THREAD

**Discussion by ssoney200** [gatsbyjs/gatsby#26648] [Help] [answered]
I'm hitting what I believe is a file limit on AWS using Gatbsy Source Wordpress Experimental and wondered if there is a way for me to do a Gatsby Clean type command when running with new build on the server side to clear the cache.

When I try to redeploy with tweaks to config to limit these files it's skipping over the files rather than removing anything already cached so I can't fix the file size limit issue and get the site built. 

Thanks in advance to anyone who can help.

**Comment by TylerBarnes** [comment]
Hi @ssoney200 , where are you running your build before deploying to AWS?

**Comment by ssoney200** [comment, marked as answer]
> Hi @ssoney200 , where are you running your build before deploying to AWS?

Hi Tyler, thanks for your reply.

After further investigation with AWS support, it appears our particular issue is with their side, and related to this issue of deploy failing right at end https://github.com/aws-amplify/amplify-console/issues/115 

Gets right to the end and then says:

2019-09-23T22:56:23 [INFO]: Updating Edge config
2019-09-23T22:56:23 [ERROR]: Failed to deploy
2019-09-23T22:56:23 [INFO]: Deployment finished.

When we first spoke to AWS they mentioned possibly size of build artefact being issue, or maybe file limit etc they didn't know for sure, but it seems to happen on AWS regardless of that. On link posted above, someone even has the issue when running 8MB build.

So, it seems like it's an ongoing issue with Amplify . Be keen to know if anyone else reading this has faced similar issues whilst running Gatsby on AWS though and if they found a solution whilst continuing to do so. 

Thanks again,
Jamie
